.NET/Microsoft.Recognizers.Text.Number/NumberRecognizer.cs

```diff
@@ -348,6 +349,34 @@ protected override void InitializeConfiguration()
                    new BaseNumberRangeParser(new TurkishNumberRangeParserConfiguration(
                                                  new BaseNumberOptionsConfiguration(Culture.Turkish, options))),
                    new Turkish.NumberRangeExtractor(options)));*/
+
+            RegisterModel<NumberModel>(

```

Comment:
Registration of the models can only happen once all tests pass, otherwise they are exposed to users. See TurkishNumberRange above as example.

Reply:
Sorry I didn't notice these were enabled in the previous review.